lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -122,13 +126,37 @@ def setup_volumes(self):
                 each["name"] = name
                 each["subPath"] = subpath
         self.runner_params['k8s_volume_mounts'] = volume_mounts
+        if self.runner_params.get('k8s_data_volume_claim'):
+            try:
+                param_claim = self.runner_params['k8s_data_volume_claim'].split(":")
+                data_claim_name = param_claim[0]
+            except Exception as e:
+                log.debug('Failed to parse `k8s_data_volume_claim` parameter in the kubernetes runner configuration')
+                raise e
+            data_volume_name = data_claim_name if "/" not in data_claim_name else data_claim_name.split("/")[0]
```

Comment:
Parsing could be moved to separate function and shared between `k8s_persistent_volume_claims` and `k8s_data_volume_claim` and `k8s_working_volume_claim`